README.md

```diff
@@ -1,29 +1,34 @@
 # Trilogy Adapter
 
-Active Record database adapter for [Trilogy](https://github.com/trilogy-libraries/trilogy)
+Ruby on Rails Active Record database adapter for [Trilogy](https://github.com/trilogy-libraries/trilogy), a client library for MySQL-compatible database servers, designed for performance, flexibility, and ease of embedding.
 
-This gem offers Trilogy support for versions of ActiveRecord prior to 7.1. Currently supports:
+This gem offers Trilogy support for versions of Active Record prior to v7.1. Currently supports:
 
-- Rails v7.0.x
-- Rails v6.1.x
-- Rails v6.0.x
+- ⚠️ Rails v7.1+ includes Trilogy support by default making this gem is unnecessary 
```

Comment:
Love this heads-up for folks :)

Feel it would be useful to remove "is", and encourage adding a comma after "default".  Ultimately then:

```
Rails v7.1+ includes Trilogy support by default, making this gem unnecessary.
```

Reply:
Oops! Just fixed. Thanks for catching that!